plugins/apps/threatbus_zmq_app/README.md

```diff
@@ -0,0 +1,167 @@
+Threat Bus App Plugin for ZMQ Protocol
+======================================
+
+<h4 align="center">
+
+[![PyPI Status][pypi-badge]][pypi-url]
+[![Build Status][ci-badge]][ci-url]
+[![License][license-badge]][license-url]
+
+</h4>
+
+A Threat Bus plugin that enables communication with any application that can
+communicate via [ZeroMQ].
+
+
+## Installation
+
+```sh
+pip install threatbus-vast
+```
+
+## Configuration
+
+The plugin uses ZeroMQ to communicate with applications, like the
+[VAST bridge](https://github.com/tenzir/threatbus/tree/master/apps/vast). The
+plugin serves three ZeroMQ endpoints to connect with. One endpoint for managing
+subscriptions (and thus snapshot requests). The other two endpoints are plainly
+for pub-sub operations.
```

Comment:
```suggestion
subscriptions (and thus snapshot requests). The other two endpoints exist
for pub-sub operations.
```

The reader wonders: one for publish and one for subscribe?

Reply:
Yes, that is how it's done with ZeroMQ.